app/views/patients/new.html.erb

```diff
@@ -4,4 +4,5 @@
   </div>
 </section>
 
+<%= button_to 'Demo Data', new_patient_path, params: fake_patient_params, method: :get, class: 'button is-info is-light' %>
```

Comment:
This button needs some padding. I think the best way would be to move it into the patient form:
```
<section class="section">
  <div class="container">
    <% if current_page? new_patient_path %>
      <div class="block">
        <%= button_to 'Demo Data',
          new_patient_path,
          params: fake_patient_params,
          method: :get,
          class: 'button is-info is-light'
        %>
      </div>
    <% end %>
    <%= form_with(model: patient) do |form| %>
...
```